tools/Normalise_Consent_Data.php

```diff
@@ -0,0 +1,276 @@
+<?php
```

Comment:
`php tools/Normalise_Consent_Data.php 
PHP Fatal error:  Interface 'Psr\Http\Server\RequestHandlerInterface' not found in /var/www/loris/php/libraries/NDB_Page.class.inc on line 29
`

Reply:
@ridz1208 run composer install for changes on major (and update to php7.2 and install required php extensions)